src/problem.cpp

```diff
@@ -431,6 +463,60 @@ std::vector<Solution> Problem::FindSolutionsImpl(Handle& handle,
     return ret;
 }
 
+std::vector<Solution>
+Problem::FindSolutionsImpl(Handle& handle,
+                           [[maybe_unused]] const FindOptions& options,
+                           std::size_t max_solutions,
+                           [[maybe_unused]] const Buffers& buffers,
+                           [[maybe_unused]] const SoftmaxDescriptor& softmax_desc) const
+{
+    auto ret = std::vector<Solution>();
+
+    auto ctx = ExecutionContext{&handle};
+
+    const softmax::ProblemDescription problem_description = AsSoftmax();
+
+    const auto algo = AlgorithmName{"Softmax"};
+
+    solver::softmax::AttnSoftmax attnSoftmaxSolver;
+    solver::softmax::Softmax regularSoftmaxSolver;
+
+    std::vector<solver::softmax::SoftmaxSolver*> solvers;
+
+    solvers.push_back(&attnSoftmaxSolver);
+    solvers.push_back(&regularSoftmaxSolver);
```

Comment:
[Maybe postponed] I think we can statically initialize `solvers`.

Reply:
@DrizztDoUrden Can you please suggest @Vsevolod1983 the technique for doing this? Thanks.

Reply:
The most basic primitive for this is https://github.com/ROCm/MIOpen/blob/9db0c265deb57b33d1a5669264157bc2700bd6d8/src/include/miopen/find_solution.hpp#L181

Reply:
This is postponable in contrast to a/b problem